extensions/historical-imagery/schema.json

```diff
@@ -63,41 +64,49 @@
           "type": "array",
           "allOf": [
             {
+              "type": "array",
               "contains": {
                 "const": "https://stac.linz.govt.nz/_STAC_VERSION_/historical-imagery/schema.json"
               }
             },
             {
+              "type": "array",
               "contains": {
                 "const": "https://stac-extensions.github.io/eo/v1.0.0/schema.json"
               }
             },
             {
+              "type": "array",
               "contains": {
                 "const": "https://stac-extensions.github.io/projection/v1.0.0/schema.json"
               }
             },
             {
+              "type": "array",
               "contains": {
                 "const": "https://stac-extensions.github.io/file/v2.0.0/schema.json"
               }
             },
             {
+              "type": "array",
               "contains": {
                 "const": "https://stac.linz.govt.nz/_STAC_VERSION_/aerial-photo/schema.json"
               }
             },
             {
+              "type": "array",
               "contains": {
                 "const": "https://stac.linz.govt.nz/_STAC_VERSION_/camera/schema.json"
               }
             },
             {
+              "type": "array",
               "contains": {
                 "const": "https://stac.linz.govt.nz/_STAC_VERSION_/film/schema.json"
               }
             },
             {
+              "type": "array",
```

Comment:
Why does 'array' need to be specified for each item?

Reply:
[The docs](https://ajv.js.org/strict-mode.html) clarify this:

> This is a very common mistake that even people experienced with JSON Schema often make - the problem here is that any value that is not an object would be valid against this schema - this is rarely intentional.

Reply:
In fact, looking at the output there are a bunch more of these. I'll do those as well, but we can put those in a separate PR if you want.

Reply:
I'm not sure i follow the docs here, do you have an example of what would be valid that shouldn't be valid?

Reply:
I can't seem to create a test which makes the lints *necessary,* so it's possible ajv is just being overzealous. There are a few issues related to how `strictTypes` works, but I couldn't see one which is strictly equivalent to not being useful. I guess this would just be a "better safe than sorry" change, which we can refine if/when ajv refines their validation.

Reply:
I wonder why its a lint issue then :shrug: ?

Reply:
Because, in general, if you don't specify the type any type is valid. If you happen to have a sufficiently loose child schema this might result in weird stuff like making a null valid in place of an object.

Reply:
In any case, this discussion is now relevant for [the other PR](https://github.com/linz/stac/pull/167). What do you think of the current PR?